helix_fhir_client_sdk/responses/fhir_response_processor.py

```diff
@@ -510,18 +515,33 @@ def get_chunk_iterator() -> (
                 if fn_handle_streaming_chunk:
                     await fn_handle_streaming_chunk(chunk_bytes, chunk_number)
                 chunk = chunk_bytes.decode("utf-8")
+                chunk_length = len(chunk_bytes)
+                total_kilobytes += chunk_length // 1024
```

Comment:
this `//` is aka `floor` division - shouldn't this be `ceil(chunk_length / 1024)` instead? 
so that we don't lose the factional remainders to calculate for enough kilobytes?

Reply:
we're just trying to get the numbers without fractions.  chunks come bigger than 1K